internal/lib/constants.go

```diff
@@ -256,6 +256,7 @@ const (
 	AviIngressController             = "ako.vmware.com/avi-lb"
 	AKOConditionType                 = "ako.vmware.com/ObjectDeletionInProgress"
 	DefaultSecretEnabled             = "ako.vmware.com/enable-tls"
+	VSTrafficEnabled                 = "ako.vmware.com/enable-traffic"
```

Comment:
I think we should call it "ako.vmware.com/disable-traffic" and set that to True to disable traffic